src/ai.rs

```diff
@@ -0,0 +1,75 @@
+use crate::board::Board;
+use crate::model::{CellContent, GameType, Stone};
+use crate::traits::GomokuAI;
+use std::path::Path;
+
+pub struct Ai {
+    pub board: Board,
+}
+
+impl Ai {
+    const ABOUT_MAP: [(&'static str, &'static str); 4] = [
+        ("name", "MyAi"),
+        ("version", "0.1"),
+        ("author", "L'eau ça rouille"),
+        ("country", "France"),
+    ];
+
+    pub fn new() -> Self {
+        Ai {
+            board: Board::new(),
+        }
+    }
+}
+
+impl GomokuAI<u8> for Ai {
+    fn start(board_size: u8) -> Option<Self> {
+        let ai = Ai::new();
+
+        if ai.board.size != board_size {
+            return None;
+        }
+        Some(ai)
+    }
+
+    fn receive_opponent_turn(&mut self, pos: &(u8, u8)) {
+        self.board.board[pos.0 as usize][pos.1 as usize] =
```

Comment:
Shouldn't we index in the opposite order? (if we receive coordinates (x, y), we should index `board[y][x]`, right?)

Reply:
yep